src/sentry/incidents/endpoints/project_alert_rule_index.py

```diff
@@ -69,8 +70,18 @@ def post(self, request, project):
 
         if serializer.is_valid():
             alert_rule = serializer.save()
+            # Parse referrer query param from referer http header
+            referer = request.META.get("HTTP_REFERER")
```

Comment:
Did you consider reading the referrer query string out in the browser code, and passing the referrer in as a request payload parameter? Doing that would avoid needing to URL parsing here and make the referrer a more formal part of the API.

Reply:
Yeah that seems like a better idea